core/backend.py

```diff
@@ -35,23 +34,16 @@ class DeepLabModel(object):
     OUTPUT_TENSOR_NAME = 'SemanticPredictions:0'
     FROZEN_GRAPH_NAME = 'frozen_inference_graph'
 
-    def __init__(self, tarball_path):
+    def __init__(self, model_path):
         """Creates and loads pre-trained deeplab model."""
         self.graph = tf.Graph()
 
-        graph_def = None
-        # Extract frozen graph from tar archive.
-        tar_file = tarfile.open(tarball_path)
-        for tar_info in tar_file.getmembers():
-            if self.FROZEN_GRAPH_NAME in os.path.basename(tar_info.name):
-                file_handle = tar_file.extractfile(tar_info)
-                graph_def = tf.GraphDef.FromString(file_handle.read())
-                break
-
-        tar_file.close()
+        # Extract frozen graph
```

Comment:
We can keep this - and do that tar extraction in Dockerfile (see other comment on tar packaging)